py_mmd_tools/nc_to_mmd.py

```diff
@@ -1045,6 +1045,37 @@ def check_conventions(self, ncin):
                     'Please provide the ACDD convention version in '
                     'the "Conventions" attribute.')
 
+    def check_feature_type(self, ncin):
+        """ Check that the featureType attribute defined in the CF
+        conventions is present in the netcdf file. Issue a warning
+        if it is not present.
+        """
+        attr = "featureType"
+        # the options could perhaps be picked up from a vocabulary
+        # service as well but this should be good enough
+        options = [
+            "point",
+            "timeseries",
+            "trajectory",
+            "profile",
+            "timeseriesprofile",
+            "trajectoryprofile"]
+        if attr not in ncin.ncattrs():
+            self.missing_attributes["warnings"].append(
+                "CF attribute %s is missing - one of the "
+                "feature types listed in Table 9.1 in https://"
+                "cfconventions.org/Data/cf-conventions/cf-conventions"
+                "-1.10/cf-conventions.html#"
+                "_features_and_feature_types should be used." % attr)
+        else:
+            ftype = ncin.getncattr(attr)
+            if ftype.lower() not in options:
```

Comment:
Her tester vi bare om det finnes, skal ikke tas med videre til mmd?

Reply:
ja, vi har ingen MMD-felt for dette

Reply:
hva var status for dette i MMD. skal det ikke utvides?

Reply:
vel, det er mulig det kommer en oppdatering, men inntil videre kan vi bare sjekke om attributten finnes i filen.

Reply:
:+1: